$OpenBSD: patch-copy_c,v 1.1 2014/03/12 21:20:58 sthen Exp $

http://dev.mutt.org/trac/changeset/9bf7593e3c08cc32bd69595d5c1cac75c29ba09d

Fix buffer overrun caused by not updating a string length after address
expansion.

--- copy.c.orig	Wed Mar 12 21:16:54 2014
+++ copy.c	Wed Mar 12 21:17:46 2014
@@ -254,6 +254,7 @@ mutt_copy_hdr (FILE *in, FILE *out, LOFF_T off_start, 
     {
       if (!address_header_decode (&this_one))
 	rfc2047_decode (&this_one);
+	this_one_len = mutt_strlen (this_one);
     }
     
     if (!headers[x])
